enews-node/app.js

```diff
@@ -0,0 +1,19 @@
+var NewsFetcher = require('./newsFetcher.js')
+
+var token = process.env.token;
+var newsFetcher = new NewsFetcher(token);
+
+var latestDate = new Date();
+var oldestDate = new Date();
+
+oldestDate.setDate(latestDate.getDate() - 31);
+
+// node timestamps are in milliseconds, need to convert to epoch
+var latest = latestDate / 1000;
+var oldest = oldestDate / 1000;
+
+newsFetcher.getEnews(oldest, latest, function(eNews) {
+  eNews.forEach(function(each) {
+    console.log(each);
```

Comment:
I couldn't think of a singular for enews
On 8 Feb 2016 9:23 a.m., "Paul Blundell" <notifications@github.com> wrote:

> In enews-node/app.js
> <https://github.com/novoda/spikes/pull/19#discussion_r52145792>:
>
> > +
> > +var token = process.env.token;
> > +var newsFetcher = new NewsFetcher(token);
> > +
> > +var latestDate = new Date();
> > +var oldestDate = new Date();
> > +
> > +oldestDate.setDate(latestDate.getDate() - 31);
> > +
> > +// node timestamps are in milliseconds, need to convert to epoch
> > +var latest = latestDate / 1000;
> > +var oldest = oldestDate / 1000;
> > +
> > +newsFetcher.getEnews(oldest, latest, function(eNews) {
> > +  eNews.forEach(function(each) {
> > +    console.log(each);
>
> each is poor variable name. What is it iterating? chat messages
> containing the hash tag right, maybe article or eNewsArticle or message
> etc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/spikes/pull/19/files#r52145792>.
>